app/scripts/app-init.js

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line import/unambiguous
+function tryImport(...fileNames) {
+  try {
+    // eslint-disable-next-line
+    importScripts(...fileNames);
+    return true;
+  } catch (e) {
+    console.error(e);
+    return false;
+  }
+}
+
+function importAllScripts() {
+  const startImportScriptsTime = Date.now();
+  // applyLavaMost has been hard coded to true as
+  // tryImport('./runtime-cjs.js') is giving issue otherwise.
+  const applyLavaMoat = true;
+
+  tryImport('./globalthis.js');
+  tryImport('./sentry-install.js');
+
+  if (applyLavaMoat) {
+    tryImport('./runtime-lavamoat.js');
+    tryImport('./lockdown-more.js');
+    tryImport('./policy-load.js');
+  } else {
+    tryImport('./lockdown-install.js');
+    tryImport('./lockdown-more.js');
+    tryImport('./lockdown-run.js');
+    tryImport('./runtime-cjs.js');
+  }
```

Comment:
Above I have hard coded `applyLavaMoat` to true as else condition is breaking in `tryImport('./runtime-cjs.js');` with this error:

![Screenshot 2022-05-23 at 5 21 07 PM](https://user-images.githubusercontent.com/2182307/169825296-49c497cf-9b87-407a-8bad-ffba20ed789e.png)

Reply:
i gues xhr is not available in service workers? its web3 junk we're not using anyways. can fix with a patch.